version/version.go

```diff
@@ -10,15 +10,15 @@ import (
 	"gopkg.in/urfave/cli.v1"
 )
 
-const version = "3.2.0"
+const Version = "3.3.0"
```

Comment:
@autrilla Doing this here as it's needed to make the fix, but not sure if this PR is the right place to actually do this. Just wanted to call it out.

Reply:
I think it's fine to do it in this PR